### DISCUSSION TITLE
redundant file?

### DISCUSSION THREAD

**Discussion by jman-blue** [adieyal/sd-dynamic-prompts#114] [General]
Not sure where to put this but It looks like there is a redundant file in ["collections/jumbo/aesthetics/colours"](https://github.com/adieyal/sd-dynamic-prompts/tree/main/collections/jumbo/aesthetics/colours) directory. It appears that[ "t" ](https://github.com/adieyal/sd-dynamic-prompts/blob/main/collections/jumbo/aesthetics/colours/t)is a copy of ["green."](https://github.com/adieyal/sd-dynamic-prompts/blob/main/collections/jumbo/aesthetics/colours/green.txt)

**Comment by adieyal** [comment]
Thanks - I've removed it
